.github/workflows/test.yml

```diff
@@ -11,12 +11,13 @@ on:
       - .github/workflows/build-docs.yml
       - CITATION.cff
       - .github/workflows/citation-cff.yml
+      - .github/workflows/build.yml
   pull_request:
     paths-ignore:
       - "docs/**"
       - .github/workflows/build-docs.yml
       - CITATION.cff
-      - .github/workflows/citation-cff.yml
```

Comment:
What's the reason for ignoring `citation-cff.yml` on push, but not on pull?

Reply:
None that I am aware of, I think they should be the same.